homeassistant/components/heos/config_flow.py

```diff
@@ -74,3 +81,24 @@ async def async_step_user(
             data_schema=vol.Schema({vol.Required(CONF_HOST, default=host): host_type}),
             errors=errors,
         )
+
+    async def async_step_reconfigure(
```

Comment:
this is very similar to the user flow, can't we reuse that one (and do a check for `self.source == SOURCE_USER / SOURCE_RECONFIGURE` if necessary)?

Reply:
Thanks for the review! I had considered that, but think is is a better user experience to show a reconfigure-specific screen with approrpiate strings (`reconfigure` step) and reusing `async_step_user` would introduce at least 3 conditional checks making it less maintainable and hard to follow.

Do you still want this changed?